web/packages/teleport/src/Assist/contexts/messages.tsx

```diff
@@ -146,12 +153,12 @@ export const remoteCommandToMessage = async (
     let avLogin = execCmd.selectedLogin;
     if (!avLogin) {
       // If the login has not been selected, use the first one.
-      avLogin = availableLogins ? availableLogins[0] : '';
+      avLogin = availableLogins ? findPreferredLogin(availableLogins) : '';
```

Comment:
Can we just sort the list and put "root logins" at the bottom instead of picking one random user?

Reply:
This takes the first non root login

Reply:
Yes, but the list is not sorted, to begin with, so "the first one" for two different API calls can be a different user. I was thinking about the following:
1. Remove the "root" login from the `availableLogins.`
2. Sort the "non-root" login (this will make the list the same between calls)
3. Append "root" logins at the end so "more likely" logins are at the top.

There are, of course, different ways of implementing it.

Reply:
I've changed this to sort the logins alphabetically and put the root logins at the end